pyproject.toml

```diff
@@ -75,6 +75,11 @@ force-exclude = '''
 [tool.ruff]
 line-length = 88
 select = ["ALL"]
+exclude=[
+    "astropy/extern/*",
+    "*_parsetab.py",
+    "*_lextab.py"
+]
```

Comment:
Was there some good reason for moving these lines around? If not then I'd recommend leaving the `exclude` where and as it was.

Reply:
A few, of varying qualities of good.
1. alphabetical.
2. I wanted ``extend-ignore`` closer to the ``per-file ignores`` as both are for check-specific ignores, while ``exclude`` ignore all checks.